synthtool/languages/java.py

```diff
@@ -92,9 +91,30 @@ def _download_formatter(version: str, dest: Path) -> None:
         fh.write(response.content)
 
 
+HEADER_REGEX = re.compile("\\* Copyright \\d{4} Google LLC")
+
+
+def _file_has_header(path: Path) -> bool:
+    with open(path, "rt") as fp:
+        for line in fp:
+            if HEADER_REGEX.search(line):
+                return True
+    return False
+
+
+# return a subset of files that do not already have a header
```

Comment:
Move this comment to be a doc string on line 107.

Reply:
Done